docs/ABOUT.md

```diff
@@ -1,3 +1,8 @@
-Scala is a object-oriented, functional language that runs on the Java Virtual Machine (JVM).  It was created by Martin Odersky.
+Scala combines object-oriented and functional programming in one concise, high-level language. Scala's static types help avoid bugs in complex applications, and its JVM and JavaScript runtimes let you build high-performance systems with easy access to huge ecosystems of libraries.
```

Comment:
I think `to huge ecosystems` should be `two huge ecosystems`

Reply:
I lifted this text from the Scala web site. I think 'to' is what they had in mind. Not the number 2.